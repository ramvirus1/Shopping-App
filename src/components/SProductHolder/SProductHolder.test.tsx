import renderer, { ReactTestRendererJSON } from 'react-test-renderer';
import SProductholder from "./index";

const props = {
    item: {
        "id": 1,
        "colour": "Black",
        "name": "Black Sheet Strappy Textured Glitter Bodycon Dress",
        "price": 10,
        "img": "http://cdn-img.prettylittlething.com/9/0/a/a/90aa90903a135ee59594f47c7685aa7ef3046e44_cly8063_1.jpg?imwidth=1024"
    },
    onSelection: (value) => {}
};

describe('SProductholder', () => {
  const snapshot = renderer.create(<SProductholder {...props} />).toJSON();
  it('should render', () => {
    expect(snapshot).toMatchSnapshot();
  });
});