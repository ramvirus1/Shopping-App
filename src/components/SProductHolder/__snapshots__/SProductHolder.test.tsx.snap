// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SProductholder should render 1`] = `
<View
  accessible={true}
  collapsable={false}
  focusable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "flexBasis": 0,
      "flexGrow": 1,
      "flexShrink": 1,
      "opacity": 1,
    }
  }
>
  <View
    accessibilityIgnoresInvertColors={true}
    style={
      Array [
        Object {
          "borderColor": "#ddd",
          "borderRadius": 4.285714285714286,
          "borderStyle": "solid",
          "borderWidth": 1,
          "flexBasis": 0,
          "flexGrow": 1,
          "flexShrink": 1,
          "height": 392.35294117647055,
          "marginBottom": 10.714285714285714,
          "marginLeft": 10.714285714285714,
          "marginRight": 10.714285714285714,
          "marginTop": 10.714285714285714,
        },
      ]
    }
  >
    <Image
      resizeMode="resize"
      source={
        Object {
          "uri": "http://cdn-img.prettylittlething.com/9/0/a/a/90aa90903a135ee59594f47c7685aa7ef3046e44_cly8063_1.jpg?imwidth=1024",
        }
      }
      style={
        Array [
          Object {
            "bottom": 0,
            "left": 0,
            "position": "absolute",
            "right": 0,
            "top": 0,
          },
          Object {
            "height": 392.35294117647055,
            "width": undefined,
          },
          undefined,
        ]
      }
    />
    <View
      style={
        Array [
          Object {
            "backgroundColor": "white",
            "bottom": 0,
            "flexBasis": 0,
            "flexGrow": 1,
            "flexShrink": 1,
            "opacity": 0.8,
            "paddingBottom": 21.428571428571427,
            "paddingLeft": 21.428571428571427,
            "paddingRight": 21.428571428571427,
            "paddingTop": 21.428571428571427,
            "position": "absolute",
            "width": "100%",
          },
        ]
      }
    >
      <Text
        numberOfLines={2}
        size={14}
        style={
          Array [
            Object {
              "fontFamily": "Inter_700Bold",
              "fontSize": 30,
            },
          ]
        }
        weight=" Inter_700Bold"
      >
        Black Sheet Strappy Textured Glitter Bodycon Dress
      </Text>
      <Text
        numberOfLines={2}
        size={16}
        style={
          Array [
            Object {
              "fontFamily": "Inter_600SemiBold",
              "fontSize": 34.285714285714285,
            },
          ]
        }
        weight="Inter_600SemiBold"
      >
        Â£ 10
      </Text>
    </View>
  </View>
</View>
`;
